docs/modules/ROOT/pages/how-to/day2ops/disable-access.adoc

```diff
@@ -0,0 +1,41 @@
+= Disable access to {product}
+
+This guide describes how to remove a user's access to all {zone}s
+
+NOTE: This page will be updated once we've implemented support for selective access to single {zone}s.
+
+== Prerequisites
+
+* Installed {idp} (see xref:appuio-cloud:root:how-to/keycloak-setup.adoc[Install Keycloak])
+* Administrator console access to {idp}
+
+== Extra steps for brokered users
```

Comment:
IMHO, it would be overkill to have a separate example page for how to disable access for our setup, so I added the VSHN specifics as `TIP` where necessary. I'm all ears for better approaches though :)

Reply:
> so what's the target audience of this documentation?

Let's target VSHN for now. We're not there yet to resell this platform, that means we iterate over such stuff when we work on that.